test/run-tests

```diff
@@ -377,8 +382,16 @@ def main():
     parser.add_argument("--cache", default="/cache",
                         help="Directory for caching VM images")
     parser.add_argument(
-        "--inventory",
+        "--inventory", help="Inventory to use for VMs",
         default='/usr/share/ansible/inventory/standard-inventory-qcow2')
+    parser.add_argument("pull_request", nargs="*", default=None,
+                        help="Pull requests to test. Example: "
```

Comment:
Is it intentional that it is `pull_request` and not `--pull_request`?

Reply:
Ah, it is intended to run like `run-tests linux-system-roles/network/50`, right?

Reply:
yes, this is the intended command line